src/app/content/hooks/receiveContent.spec.ts

```diff
@@ -67,6 +67,16 @@ describe('setHead hook', () => {
     expect(dispatch).not.toHaveBeenCalledWith(setHead(expect.anything()));
   });
 
+  it('does nothing if page is reloading', async() => {
+    store.dispatch(receiveBook(combinedBook));
+    store.dispatch(receivePage({...page, references: []}));
+    store.getState().content.loading = {page: {slug: 'pageId'}};
```

Comment:
This was needed to restore coverage to this line: https://github.com/openstax/rex-web/blob/main/src/app/content/hooks/receiveContent.ts#L35

I have no idea how it was covered before, since I don't think any of the things we can dispatch seem to set `loadingPage` without clearing `page`.